pydantic/dataclasses.py

```diff
@@ -86,6 +86,8 @@ def __validate__(cls: Type['DataclassT'], v: Any) -> 'DataclassT':
     'make_dataclass_validator',
 ]
 
+_T = TypeVar('_T')
```

Comment:
```suggestion
T = TypeVar('T')
```
Unless we need the leading underscore?

Reply:
i usually do that with `TypeVar`s and other variables/functions to indicate that they are "private" ie. shouldn't be imported elsewhere.

not sure what the standard is so i'll just remove it if you want

Reply:
looks like there are already `TypeVar`s in this repo following the same pattern i do

https://github.com/pydantic/pydantic/blob/fa6b9788a1001591cbac44923476b5d12a14c1fc/pydantic/main.py#L95

Reply:
okay, that's fine.